slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtection.java

```diff
@@ -251,7 +251,11 @@ public void prune() {
     LOG.info("Pruning slashing protection database");
     registeredValidators
         .values()
-        .forEach(v -> dbPruner.pruneForValidator(v, pruningEpochsToKeep, pruningSlotsPerEpoch));
+        .forEach(
+            v -> {
+              LOG.debug("Pruning for validator {}", registeredValidators.inverse().get(v));
```

Comment:
nit: `() -> registeredValidators.inverse().get(v)` would allow it to be execute only if debug level is enabled.

Reply:
done